conf/messages.cy

```diff
@@ -79,5 +79,11 @@ subpage.paymentsnotice = Bydd taliadau’n cymryd 4 i 7 diwrnod gwaith i ymddang
 account.summary.fileareturn = Llenwch eich Ffurflen
 account.summary.makeAPayment = Gwnewch daliad<span class="visuallyhidden"> Treth Gorfforaeth</span>
 
+account.summary.indebit = Mae arnoch {0} -
+account.summary.incredit = Rydych {0} mewn credyd -
+account.summary.nothingtopay = Does dim byd gennych i’w dalu -
+account.summary.seebreakdown = gweler y dadansoddiad
+account.summary.viewstatement = gweld y datganiad
 account.summary.nobalance = Dim gwybodaeth am weddill i’w dangos
-account.summary.genericerror = Ni allwn ddangos eich gwybodaeth Treth Gorfforaeth ar hyn o bryd.
\ No newline at end of file
+account.summary.genericerror = Ni allwn ddangos eich gwybodaeth Treth Gorfforaeth ar hyn o bryd.
+account.summary.genericerror.portal = Try refreshing the page in a few minutes or use the {0}old HMRC website{1}.
```

Comment:
Can this be considered done as it is, or do we have to wait for this line to be translated?

Reply:
I must have addressed this while you were reviewing 😅